web/app/components/comment/__title/comment__title.scss

```diff
@@ -0,0 +1,17 @@
+.comment__title {
+  margin-bottom: 0.2rem;
+
+  &-link {
```

Comment:
And also, don't write code in this way, please. Use full selectors. Right know it's impossible to find `__title-link` using IDE search.

Reply:
ok, I get it.